tool/lib/devtools_command_runner.dart

```diff
@@ -38,5 +42,25 @@ class DevToolsCommandRunner extends CommandRunner {
     addCommand(UpdateDevToolsVersionCommand());
     addCommand(UpdateFlutterSdkCommand());
     addCommand(UpdatePerfettoCommand());
+
+    argParser.addFlag(
+      _flutterFromPathFlag,
+      abbr: 'p',
+      negatable: false,
+      help: 'Use the Flutter SDK on PATH rather than the one being used to run '
+          'devtools_tool (usually tool/flutter-sdk)',
```

Comment:
nit this makes it sound like the tool/flutter-sdk is still being used to run `devtools_tool` even if the flutter from path flag is passed. Maybe reword: "Use the Flutter SDK on PATH to run devtools_tool commands and sub commands, instead of the Flutter SDK from tool/flutter-sdk which is used by default."

Reply:
Does my update make sense? Something that's slightly odd but I don't think matters enough to mention (because it makes things more verbose) is that this only applies to the spawned processes - the current process is using whatever the shell script decided to use (because we're already dunning a VM by the time we parse this flag).

I did consider once having the top level runner detect if this flag was set but it was running with a different SDK, and just re-run itself with the new one, but I'm not sure there would be any real benefit to it.

Reply:
If the only thing that could be run with a different SDK is the entrance to this script, then I think this is fine because all sub commands will respect the Flutter SDK we select.

Reply:
Yep, that's correct - as long as we don't add new code that accidentally runs things from path instead of using the current SDK, but as long as we go through `CliCommand.flutter` etc., then this should be hard to do accidentally.